syntax="proto2";
option cc_generic_services = true;

package keyvi_server.service;

message GetRequest {
      required string key = 1;
};

message GetResponse {
      required string value = 1;
};

message SetRequest {
      required string key = 1;
      required string value = 2;
};

message SetResponse {
};

service Index {
      rpc Get(GetRequest) returns (GetResponse);
      rpc Set(SetRequest) returns (SetResponse);
};
